### PR TITLE
removes dead tor code

### PR DESCRIPTION
**Pull request by harryob**
this isn't enabled on the live server, nor would it ever be

:cl:
server: tor banning functionality is removed
/:cl:

### CODE REVIEW

**Comment by SabreML** [comment]
There's config section for this too:
https://github.com/cmss13-devs/cmss13/blob/7846f4194a36d4ba0eb5b7851b82c0df1c5fa874/config/example/config.txt#L154-L155

**Comment by harryob** [comment]
> There's config section for this too:
> 
> https://github.com/cmss13-devs/cmss13/blob/7846f4194a36d4ba0eb5b7851b82c0df1c5fa874/config/example/config.txt#L154-L155

it is no more

File: code/modules/admin/IsBanned.dm [hunk 1]
```diff
@@ -26,19 +27,8 @@
...
-		message_admins("Failed Login: [src] - Banned: ToR")
-		return list("reason"="Using ToR", "desc"="\nReason: The network you are using to connect has been banned.\nIf you believe this is a mistake, please request help at [CONFIG_GET(string/banappeals)]")
-
-	// wait for database to be ready
 
 	. = P.check_ban(computer_id, address)
-	if(.)
-		return .
-
-	return ..() //default pager ban stuff
```
Comment (Drulikar) [inline code/modules/admin/IsBanned.dm:41]: Are you sure this is related and you do want to change?

Comment (harryob) [reply code/modules/admin/IsBanned.dm:41]: escaped from the atomisation chamber of #5628

**Review by Drulikar — APPROVED** [review]

**Pull request merged**